examples/src/test/java/io/camunda/zeebe/process/test/examples/PullRequestProcessTest.java

```diff
@@ -225,12 +225,33 @@ private void waitForBusyState(final Duration duration)
   private PublishMessageResponse sendMessage(
       final String messageName, final String correlationKey, final Map<String, Object> variables)
       throws InterruptedException, TimeoutException {
+
+    /*
+     To avoid flaky tests, we recommend publishing messages without a time to live when using time
+     manipulation in the same test case. Alternatively, you could plan out the timings of your time
+     manipulation and the published message's expiry.
+
+     In these tests, we assume that a timer event has triggered after the {@code increaseTime}
+     method returns. However, this is not guaranteed if a time to live is set because the message
+     could expire. Depending on the time to live, the message can expire due to time manipulation.
+
+     The {@code increaseTime} method will return after waiting for the engine to become idle again.
+     However, message expiry can cause the engine to be busy followed by being idle again. So, the
+     increaseTime method can return before the timer event has triggered when a message expires
+     due to time manipulation. This can be the cause of a flaky test.
+
+     Note that by default, the time to live is set to 1 hour.
+     See {@code ZeebeClientBuilder#defaultTimeToLive}.
+    */
```

Comment:
💭 I was wondering if it makes sense to have such a comment in the utility class as well. Either just like it is or in a shorter form. In the end, before/when using the utility method, it would be nice to be made aware of this. What do you think?

Reply:
👍 I don't think the entire comment is necessary, but it would help to add JavaDoc that it publishes a message with TTL 0 and that an overloaded method exists to set a specific TTL. Note that the Utilities class is internal test only (it exists in test resources and is not something our users can use).

Reply:
@tmetzke I've added a commit, could you have another look?